src/track/beatutils.h

```diff
@@ -68,6 +68,15 @@ class BeatUtils {
         const QVector<double> rawbeats, const int sampleRate,
         const int totalSamples, const double globalBpm);
 
+    // TODO(JVC) Document
+    static constexpr int kFrameSize = 2;
```

Comment:
The term *frame size* is misleading. You are actually referring to the number of channels.

Why always 2 channels? If this is bound to our engine then there is already a constant: `mixxx::kEngineChannelCount`

Reply:
Comes from previous code. I have done the change, but it involved including engine/engine.h only to get that constant and that looks ugly.
May be we must have a separate header for general use constants? This will make it cleaner.